tests/test_asgi.py

```diff
@@ -878,7 +883,8 @@ async def app(websocket: WebSocket) -> None:
 
     client = TestClient(app)
     with pytest.raises(WebSocketDisconnect) as exc:
-        client.websocket_connect("/")
+        with client.websocket_connect("/") as _:
```

Comment:
Why not delete `as _`?

Reply:
This was also related to the websocket code. I removed the change and everything tested fine.